arrow-site/docs/docs/fx/README.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: docs-fx
-title: Fx
+title: Arrow Fx. Typed FP for the masses
```

Comment:
```suggestion
title: Arrow Fx. Async and Concurrent Functional Programming for Kotlin
```

Reply:
Should probably change other references or look for a better phrase or tagline in this context. FP for the masses sounds empty these days